packages/core-api/src/apis/implementations/auth/github/GithubAuth.ts

```diff
@@ -50,13 +50,16 @@ const DEFAULT_PROVIDER = {
 };
 
 class GithubAuth implements OAuthApi, SessionApi {
-  static create({
-    discoveryApi,
-    environment = 'development',
-    provider = DEFAULT_PROVIDER,
-    oauthRequestApi,
-    defaultScopes = ['read:user'],
-  }: OAuthApiCreateOptions) {
+  static create(
+    {
+      discoveryApi,
+      environment = 'development',
+      provider = DEFAULT_PROVIDER,
+      oauthRequestApi,
+      defaultScopes = ['read:user'],
+    }: OAuthApiCreateOptions,
+    storageKey = 'githubSession',
```

Comment:
Sorry, big nope on adding this as a separate arg so rejecting to make sure we don't merge this form